src/test/java/Homework/DropdownCheckboxAlertTest.java

```diff
@@ -0,0 +1,76 @@
+package Homework;
```

Comment:
move to package `theinternet`
also the package name should be in all lower cases

Reply:
resolved